test/views/vat/WhatIsYourVATRegNumberViewSpec.scala

```diff
@@ -68,6 +68,12 @@ class WhatIsYourVATRegNumberViewSpec extends ViewBehaviours {
 
         assertEqualsMessage(doc, "title", "error.browser.title", title)
       }
+
+      "show error summary when there is error " in {
+        val doc = asDocument(createViewUsingForm(form.bind(Map("value" -> ""))))
+
+        assertRenderedById(doc, "error-summary-heading")
```

Comment:
please test the error message is correct

Reply:
@suyangyu  : Its done. Thanks :-)